### PR TITLE
EBODAC-894 - Write Test For Capacity Limitation

### PR DESCRIPTION
**Pull request by MikolajBasSoldevelo**

### CODE REVIEW

File: booking-app/src/test/java/org/motechproject/bookingapp/service/ScreeningServiceTest.java [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+
+    @Before
+    public void setUp() {
+        initMocks(this);
+        clinic = new Clinic("siteId", "location", 1, 5, 2, 9, 8, 7, 10, 10, 10, 10, 10, 10, 10);
+        clinic.setId(1l);
+    }
+
+    @Test
+    public void shouldThrowNumberOfRoomsExceptionWhenNumberOfRoomsLimitIsReached() {
```
Comment (pmuchowski) [inline booking-app/src/test/java/org/motechproject/bookingapp/service/ScreeningServiceTest.java]: we do not have NumberOfRoomsException, only LimitationExceededException, change the name of this test and all other tests with non existing exceptions in the name

**Comment by MikolajBasSoldevelo** [comment]
test this pleae

**Comment by pmuchowski** [comment]
test this please

**Comment by pmuchowski** [comment]
test this please

**Pull request merged**